docs/examples/window/ClickAway/README.md

```diff
@@ -1 +1 @@
-undefined
\ No newline at end of file
+A container that listens for clicks outside of the children. Useful for having components that hide themselves when the user clicks on the rest of the app (such as in a `Select`).
```

Comment:
A container that listens for clicks outside of ~~the~~ its children. Useful **in cases where** ~~for having~~ components ~~that~~ hide themselves when the user clicks on the rest of the app (such as **outside the confines of** a `Select`).